x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -489,6 +499,43 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
     ]
   );
 
+  const GroupByChildren = useCallback(
+    ({ groupByRadioSelection, groupByDurationUnit, groupByDurationValue }) => (
+      <EuiRadioGroup
+        disabled={
+          !license.isAtLeast(minimumLicenseForSuppression) ||
+          groupByFields == null ||
+          groupByFields.length === 0
+        }
+        idSelected={groupByRadioSelection.value}
+        options={[
+          {
+            id: GroupByOptions.PerRuleExecution,
+            label: 'Per rule execution',
+          },
+          {
+            id: GroupByOptions.PerTimePeriod,
+            label: (
+              <>
+                {`Per time period`}
+                <DurationInput
+                  durationValueField={groupByDurationValue}
+                  durationUnitField={groupByDurationUnit}
+                  isDisabled={groupByRadioSelection.value !== GroupByOptions.PerTimePeriod}
```

Comment:
Itty bitty nit: Looks like the the `Per time period` control doesn't go back to being disabled (just the radio buttons) when the suppression field input is cleared:


<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/213580907-bca421c0-240e-4612-bb65-2af93ae663db.gif" />
</p>

Reply:
This should do it:

```suggestion
                  isDisabled={groupByFields?.length === 0 || groupByRadioSelection.value !== GroupByOptions.PerTimePeriod}
```